k8s/demo/common/ccd/ccd-demo.yaml

```diff
@@ -52,12 +63,47 @@ spec:
         importerPassword:
           secretKeyRefName: ccd-importer-password
           secretKeyRefKey: key
-        redirectUri: 'https://case-management-web-ccd-cmc-citizen-frontend.demo.platform.hmcts.net/oauth2redirect'
+        redirectUri: 'https://case-management-web-ccd.demo.platform.hmcts.net/oauth2redirect'
+    
+
+  #CCD Front-end
+    caseManagementWeb:
+      environment:
+        IDAM_LOGIN_URL: https://idam-api.demo.platform.hmcts.net
+
+  #Dependencies
+    draft-store-service:
+      java:
+        releaseNameOverride: ccd-demo-draftstore
+        environment:
+          IDAM_URL: https://idam-api.demo.platform.hmcts.net
+          S2S_URL: http://ccd-demo-s2s
+          DRAFT_STORE_DB_HOST: ccd-demo-postgres
     rpe-service-auth-provider:
       java:
         releaseNameOverride: ccd-demo-s2s
+
+    payment-api:
```

Comment:
Where's the corresponding change to chart-ccd to add this?
I would expect to see the other services here:
https://github.com/hmcts/chart-ccd/blob/master/ccd/requirements.yaml

https://github.com/hmcts/chart-cmc/blob/master/cmc/requirements.yaml#L25

Reply:
we have paymentsUrl in Values.yaml  and in turn paymentsUrl is used in api-gateway.yaml template. hence i added this  payment-api: